packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.styles.ts

```diff
@@ -6,17 +6,23 @@ const GlobalClassNames = {
 };
 
 export const getStyles = (props: ISwatchColorPickerStyleProps): ISwatchColorPickerStyles => {
-  const { className, theme } = props;
+  const { className, theme, cellMargin = 10 } = props;
```

Comment:
Same... move this to the `defaultProps` static member of the `SwatchColorPicker` class